code/modules/hydroponics/grown.dm

```diff
@@ -1119,15 +1118,71 @@ obj/item/weapon/reagent_containers/food/snacks/grown/shell/eggy/add_juice()
 
 /obj/item/weapon/reagent_containers/food/snacks/grown/tea/aspera
 	seed = /obj/item/seeds/tea_aspera_seed
-	name = "Tea Aspera tips"
+	name = "tea aspera tips"
 
 
 /obj/item/weapon/reagent_containers/food/snacks/grown/tea/astra
 	seed = /obj/item/seeds/tea_astra_seed
-	name = "Tea Astra tips"
+	name = "tea astra tips"
 	desc = "These aromatic tips of the tea plant can be dried to make tea."
 	icon_state = "tea_astra_leaves"
 
 /obj/item/weapon/reagent_containers/food/snacks/grown/tea/astra/add_juice()
 	..()
-	reagents.add_reagent("kelotane", 1 + round((potency / 20), 1))
\ No newline at end of file
+	reagents.add_reagent("kelotane", 1 + round((potency / 20), 1))
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/panana
+	seed = /obj/item/seeds/pananaseed
+	name = "panana"
+	desc = "It's an excellent prop for a mime."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "panana"
+	trash = /obj/item/weapon/grown/bananapeel/specialpeel/pananapeel
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/panana/add_juice()
+	if(..())
+		reagents.add_reagent("nothing", 1 + round((potency / 10), 1))
+		reagents.add_reagent("mutetoxin", 1 + round((potency / 10), 1))
+
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/singulopotato
+	seed = /obj/item/seeds/singulopotatoseed
+	name = "singularity potato"
+	desc = "This strange potato seems to have its own gravitational pull..."
+	icon_state = "singulopotato"
+	dried_type = /obj/item/weapon/reagent_containers/food/snacks/grown/potato
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/singulopotato/add_juice()
+	if(..())
+		reagents.add_reagent("nutriment", 1 + round((potency / 10), 1))
+		reagents.add_reagent("vitamin", 1 + round((potency / 10), 1))
+		reagents.add_reagent("singulo", 1 + round((potency / 10), 1))
+		bitesize = reagents.total_volume
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/singulopotato/attack_self(mob/user as mob)
+	user.visible_message("<span class='warning'>[user] crushes a singularity potato!</span>")
+	var/vortexpower = max(3, round(potency/14, 1)) // this limits the range to 7 at max potency.
+	var/turf/pull = user.loc
+	playsound(user, 'sound/effects/empulse.ogg', 50, 1)
+	for(var/atom/X in orange(vortexpower,pull))
+		if(istype(X, /atom/movable))
+			if(X == user) continue
+			if((X) &&(!X:anchored) && (!istype(X,/mob/living/carbon/human)))
+				step_towards(X,pull)
+				step_towards(X,pull)
+				step_towards(X,pull)
+				step_towards(X,pull)
+			else if(istype(X,/mob/living/carbon/human))
+				var/mob/living/carbon/human/H = X
+				if(istype(H.shoes,/obj/item/clothing/shoes/magboots))
+					var/obj/item/clothing/shoes/magboots/M = H.shoes
+					if(M.magpulse)
+						continue
+				H.apply_effect(min(3, vortexpower))
+				step_towards(H,pull)
+				step_towards(H,pull)
+				step_towards(H,pull)
+				step_towards(H,pull)

```

Comment:
Can you change these multiple step_towards to a for loop? Just makes it look nicer.